cluster/src/main/java/org/apache/iotdb/cluster/coordinator/Coordinator.java

```diff
@@ -442,9 +445,11 @@ private TSStatus forwardToMultipleGroup(Map<PhysicalPlan, PartitionGroup> planGr
     }
     TSStatus status;
     if (errorCodePartitionGroups.isEmpty()) {
-      status = StatusUtils.OK;
       if (allRedirect) {
-        status = StatusUtils.getStatus(status, endPoint);
+        status = new TSStatus(TSStatusCode.NEED_REDIRECTION.getStatusCode());
```

Comment:
StatusUtils doesn't have a suitable static method. Although I can use `StatusUtils.getStatus(new TSStatus(TSStatusCode.NEED_REDIRECTION.getStatusCode()), endPoint)`, it will call one more deepClone(). I prefer not to do this only for save one line.

Reply:
As you wish.